app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/Sub02Controller.scala

```diff
@@ -94,6 +94,9 @@ class Sub02Controller @Inject() (
       Future.successful(Ok(xiEoriGuidancePage()))
   }
 
+  def subscriptionNextSteps(service: Service): String =
```

Comment:
Do we need a function for this since it is only used once? Not going to block the ticket since it's personal preference but just wanted to make a note